src/ext/moderation.py

```diff
@@ -29,12 +29,17 @@ async def ban(
         ),
     ):
         try:
-            if clear_messages == False:
-                await interaction.guild.ban(user=user, reason=reason, delete_message_seconds=0)
+            if interaction.user == user:
+                await interaction.send(f"{interaction.user.mention}, you can't ban yourself!", ephemeral=True)
```

Comment:
skoro interaction.user == user to wystarczy 
```suggestion
            if interaction.user == user:
                await interaction.send(f"{user.mention}, you can't ban yourself!", ephemeral=True)
```